dev/Repeater/ItemsRepeater.idl

```diff
@@ -106,11 +106,7 @@ unsealed runtimeclass ItemsSourceView: Windows.UI.Xaml.Interop.INotifyCollection
     Boolean HasKeyIndexMapping{ get; };
     String KeyFromIndex(Int32 index);
     Int32 IndexFromKey(String key);
-
-    [WUXC_VERSION_PREVIEW]
-    {
-        Int32 IndexOf(IInspectable item);
-    }
+    Int32 IndexOf(IInspectable item);
```

Comment:
There are two copies of IndexOf in the implementation which we had to keep for stable release, we should be able to remove one of them now.

Reply:
Totally missed that. Removed that now.